projects/ad777x_iio/scripts/requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy==1.22.4
+pyadi-iio==0.0.12
```

Comment:
nit: Shall we update this to the latest versions like 15?

Reply:
Yes, the change has been done